modules/core/04-channel/keeper/upgrade.go

```diff
@@ -485,9 +485,7 @@ func (k Keeper) writeUpgradeTimeoutChannel(
 		panic(fmt.Sprintf("could not find existing upgrade when cancelling channel upgrade, channelID: %s, portID: %s", channelID, portID))
 	}
 
-	if err := k.AbortUpgrade(ctx, portID, channelID, types.NewUpgradeError(channel.UpgradeSequence, types.ErrUpgradeTimeout)); err != nil {
-		return errorsmod.Wrapf(types.ErrUpgradeRestoreFailed, "err: %v", err)
-	}
+	k.restoreChannel(ctx, portID, channelID, channel)
```

Comment:
use restore here because we don't need to set an error receipt, just restoring channel state.